bpf/lib/generic.h

```diff
@@ -49,7 +49,8 @@ struct msg_generic_kprobe {
 	unsigned long a0, a1, a2, a3, a4;
 	long argsoff[MAX_POSSIBLE_ARGS];
 	struct msg_selector_data sel;
-	__u32 idx;
+	__u32 idx; /* attach cookie index */
+	__u32 index; /* recursion index for filter_read_arg/generic_process_event */
```

Comment:
This is too generic of a name. How about `filter_tailcall_index`?

Reply:
ok